chapters/limits.txt

```diff
@@ -3010,4 +3013,23 @@ respectively, otherwise both members must: be `0`.
 
 endif::VK_EXT_sample_locations[]
 
+ifdef::VK_EXT_custom_border_color[]
 
+[open,refpage='VkPhysicalDeviceCustomBorderColorPropertiesEXT',desc='Structure describing whether custom border colors can be supported by an implementation',type='structs']
+--
+The sname:VkPhysicalDeviceCustomBorderColorPropertiesEXT structure is defined as:
+
+include::{generated}/api/structs/VkPhysicalDeviceCustomBorderColorPropertiesEXT.txt[]
+
+The members of the sname:VkPhysicalDeviceCustomBorderColorPropertiesEXT structure
+describe the following features:
+
+  * [[limits-maxUniqueCustomBorderColors]] pname:maxUniqueCustomBorderColors indicates
+    the maximum number of custom border colors the application may provide. If
```

Comment:
This is named "max*Unique*CustomBorderColors", but the rest of the language doesn't use the word "unique", which feels a little contradictory.